gem/lib/pupperware/spec_helper.rb

```diff
@@ -364,6 +364,65 @@ def wait_for_pe_orchestration_services()
     end
   end
 
+  ######################################################################
+  # PE Bolt Server Helpers
+  ######################################################################
+
+  def curl_job_number(job_number, timeout = 30)
```

Comment:
I've been wondering if we should switch over to using Ruby named arguments everywhere?

We're a bit inconsistent in this file.

Reply:
Sounds good, I went ahead and updated this PR so we don't make the inconsistencies grow.